requirements.txt

```diff
@@ -22,4 +23,4 @@ tensorboard
 tensorflow
 tensorflow-probability
 wheel
-xarray[io]
+xarray[io]<2013.2.0
```

Comment:
Sorry James, still a typo, should be `xarray[io]<2023.2.0`, change being the year.

Reply:
Thanks @bnubald, just checking you're on it 😛